__tests__/integration.test.js

```diff
@@ -263,3 +263,94 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/:article_id", () => {
+  it("PATCH 200: returns the updated article with the incremented votes by the given amount.", () => {
+    const votes = { incVotes: 5 };
```

Comment:
the ticket asks for `inc_votes` - will cause a bit of confusion on the front end if you've put it as a different key as expected.